src/blog/oilpan-library.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Oilpan Library'
+author: 'Anton Bikineev, Omer Katz ([@omerktz](https://twitter.com/omerktz)), and Michael Lippautz ([@mlippautz](https://twitter.com/mlippautz)), efficient and effective file movers'
+avatars:
+  - anton-bikineev
+  - omer-katz
+  - michael-lippautz
+date: 2021-11-09
+tags:
+  - internals
+  - memory
+  - cppgc
+description: 'In V8 v9.4 we are shipping Oilpan, a garbage collection library for C++.'
```

Comment:
I think it's more correct to say "we've shipped" here as it's been some time ago now? Alternatively, perhaps at this point we shouldn't mention V8 version in the meta description and shift the focus to the library itself.

Reply:
Good point, I rephrased this a little removing the V8 version.